packages/@aws-cdk/toolkit-lib/lib/api/drift/drift-formatter.ts

```diff
@@ -172,19 +173,18 @@ export class DriftFormatter {
       unchanged += this.printSectionFooter();
     }
 
-    // Process all unchecked resources
-    if (this.allStackResources) {
```

Comment:
This is a JS Map, so always truthy. Removed.

Reply:
Nice!